HW8/src/com/jvprogram/exrcutorService/ExecutorServiceDemo.java

```diff
@@ -0,0 +1,77 @@
+package com.jvprogram.exrcutorService;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class ExecutorServiceDemo {
+
+    SimpleDateFormat sdf = null;
+    private final int COUNT = 5;
+
+    ExecutorServiceDemo () {
```

Comment:
it is not a good decision to write all logic in constructor
better write it in main